internal/support/scheduler/loader.go

```diff
@@ -25,19 +25,22 @@ func AddSchedulers() error {
 	// load data from core-metadata
 	err := loadCoreMetadataInformation()
 	if err != nil {
-		return LoggingClient.Error("failed to load information from core-metadata", err.Error())
+		LoggingClient.Error("failed to load information from core-metadata", err.Error())
```

Comment:
Should this (and other statements like it) be

`LoggingClient.Error(fmt.Sprintf("failed to load information from core-metadata %s", err.Error()))`

As this is written in the PR branch I believe it won't actually log anything due to mismatched keys to values.